lib/practice_exercises.rb

```diff
@@ -1,13 +1,59 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: ? O(n) The time is based on input and is linear.
+# Space Complexity: ? O(n) Still confused! But, the space use is not constant so it is O(n)
 def is_palindrome(string)
```

Comment:
👍  You are right on space complexity because you create new arrays with split, and select.